docs/quickstart.md

```diff
@@ -53,6 +53,117 @@ qc_mapped_level = get_benchmark(
     level=BenchmarkLevel.MAPPED,
     circuit_size=5,
     target=get_device("ionq_forte_36"),
+    opt_level=2,
 )
 qc_mapped_level.draw(output="mpl")
 ```
+
+## Mirror Circuits
+
+```{code-cell} ipython3
+qc_mirrored = get_benchmark(
+    benchmark="ghz",
+    level=BenchmarkLevel.MAPPED,
+    circuit_size=3,
+    target=get_device("ibm_falcon_27"),
+    opt_level=0,
+    # generate_mirror_circuit=True,
+)
+qc_mirrored.draw(output="mpl")
+```
+
+## Self-defined Circuits
+
+```{code-cell} ipython3
+from qiskit import QuantumCircuit
+
+qc = QuantumCircuit(2)
+qc.h(0)
+qc.cx(0, 1)
+
+qc_circuit = get_benchmark(benchmark=qc, level=BenchmarkLevel.NATIVEGATES, target=get_device("ionq_forte_36"))
```

Comment:
Although this works, I would prefer to use a native gateset target instead of a device for the native gates level.